### DISCUSSION TITLE
What happens to data sent to https://edit-csv.net/ ?

### DISCUSSION THREAD

**Discussion by PtitGuillaume** [janisdd/vscode-edit-csv#166] [General]
Hi, 
 first of all, thank you for the app! It's great, easy to use  and powerful.
I just wonder what happens to the data that are been sent on the Web-App?  

Thnank you and Kind Regards

**Comment by janisdd** [comment]
Hi, great question!

Nothing actually happens to the data in the web application. Once the page is loaded, you can switch off the Internet and it will still work (even before you load your data).

The only "problem" is that I use Google Analytics to track the usage of the web app, and I don't know if Google does anything with the content of the page (but other websites would have the same problem).

And here's a quick start on how to see if the site is sending data (while the internet is on):

- Open the ‘Developer Tools’ in your browser (the shortcut depends on the browser and your operating system)
- Switch to the ‘Network’ tab
- Enter the URL `https://edit-csv.net/` in the address bar
- You will see which files/scripts/images/... are loaded in a list
- If the website sends/requests further data, a new entry will be added to the list
  - (there is an exception (websockets) that can send data without a new entry appearing in the list)
